utils/modify-payload-directory.sh

```diff
@@ -30,7 +33,7 @@ echo -e "${GREEN_FG}[\U2714] Updated name of payload directory to:${RESET}\
 if sudo docker ps | grep 'apache' > /dev/null; then
   echo "[*] Restarting apache container"
 
-  sudo docker restart apache
+  sudo docker restart apache cobalt
```

Comment:
Is there a reason we aren't using `docker-compose`/`docker compose`?

Reply:
No reason other than me not knowing compose also had those commands. 😄 

Is there a difference between ```docker restart CONTAINER(s)``` vs ```docker-compose restart CONTAINER(s)``` ? Should we be using ```docker-compose``` instead of ```docker``` for these 2 commands since they are started with ```docker-compose```/```docker compose``` ?

Reply:
I'm not sure what the specific differences are but since this is a Compose configuration I would prefer `docker-compose`/`docker compose` for all management so that any "behind the scenes" tasks that may be handled for us are covered.

Reply:
Resolved in [ba09100](https://github.com/cisagov/megazord-composition/pull/49/commits/ba09100eb8b671add3e275845b5d831bea1a43ee).